velox/exec/Task.cpp

```diff
@@ -332,8 +332,28 @@ bool Task::allNodesReceivedNoMoreSplitsMessageLocked() const {
   return true;
 }
 
+const std::string& Task::getOrCreateSpillDirectory() {
+  if (spillDirectoryCreated_ || spillDirectory_.empty()) {
```

Comment:
you might want to annotate spillDirectoryCreated_ as tsan_atomic to avoid the fake tsan warning? thanks!

Reply:
as discussed, we are keeping this unchanged since its already declared as an std::atomic